mk/install.mk

```diff
@@ -1,11 +1,13 @@
 .PHONY: install
-install:
-	npm install
+install: check/node ## Dev: install all dependencies
+	@npm install
```

Comment:
Should we install a specific version?

Reply:
A specific version of `node`?

At the moment at least we just warn and error out if you don't have the right version of node installed: 

https://github.com/kumahq/kuma-gui/pull/2529/files#diff-c6eeb169250244938a6aee582aa3134148d4c40d1a8f18ab793e67a735f166b8R6

It takes the pinned version from here

https://github.com/kumahq/kuma-gui/blob/master/.nvmrc

At the moment its up to the engineer to then make sure the correct version is installed (usually its calling `nvm`) but it could also be `asdf` or something else